nilearn/_utils/testing.py

```diff
@@ -113,12 +113,10 @@ def serialize_niimg(img, gzipped=True):
             return f.read()
 
 
-@contextlib.contextmanager
-def write_tmp_imgs(*imgs, **kwargs):
-    """Context manager for writing Nifti images.
+def write_tmp_imgs(*imgs, file_path=None, **kwargs):
```

Comment:
Maybe the name should also be changed to something like `write_fake_imgs`

Reply:
agreed